extension/js/background.js

```diff
@@ -135,15 +136,15 @@ function handleMessage(data, sender, sendRespones) {
         timeIndex.push(time.toString());
```

Comment:
I don't see the point of changing time toString(), numbers are as good (maybe there's a reason I can't understand)